packages/rule-components/src/ReportDetails/RuleFeedback.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RuleFeedback = ruleId => (
+    <div>
+        <div>Is this rule helpful?</div>
+        {/* TODO: add like/dislike buttons here */}
```

Comment:
No need to add comments, when the implementation progresses a bit it will be added here.

Reply:
If you are wondering where to get these buttons you should import
```JS
import ThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/thumbs-up-icon';
import ThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/thumbs-down-icon';
```

Reply:
they look like this 
![image](https://user-images.githubusercontent.com/57761827/76423069-8678b500-63a6-11ea-8f2a-3fa259789b11.png) which is not similar to ones in mock up

Reply:
added ones from font awesome

Reply:
Oh, there are outlined icons? Then you can use `import ThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-up-icon';` and `import ThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-down-icon';`

Reply:
You can check the full list of icons here https://patternfly-react.surge.sh/icons